configure.ac

```diff
@@ -578,6 +578,29 @@ if test "$WITH_ECONF" = "check"; then
 elif test "$WITH_ECONF" = "yes"; then
   PKG_CHECK_MODULES([ECONF], [libeconf >= 0.5.0], [ECONF_CFLAGS="-DUSE_ECONF=1 $ECONF_CFLAGS"], [])
 fi
+
+AC_MSG_CHECKING(for econf_readConfig)
+old_libs=$LIBS
+LIBS="$LIBS -leconf"
```

Comment:
Please use `ECONF_LIBS` instead of `-leconf`.
Please also add `ECONF_CFLAGS` to `CFLAGS` for the test.

Reply:
> Please use `ECONF_LIBS` instead of `-leconf`. Please also add `ECONF_CFLAGS` to `CFLAGS` for the test.
done


> Please use `ECONF_LIBS` instead of `-leconf`. Please also add `ECONF_CFLAGS` to `CFLAGS` for the test.
Hm, where do you expect it ?

Reply:
> > Please use `ECONF_LIBS` instead of `-leconf`. Please also add `ECONF_CFLAGS` to `CFLAGS` for the test.
> 
> Hm, where do you expect it ?


e.g.
```
LIBS="$LIBS $ECONF_LIBS"
saved_CFLAGS=$CFLAGS
CFLAGS="$CFLAGS $ECONF_CFLAGS"
...
CFLAGS=$saved_CFLAGS
LIBS=$old_libs
```

Reply:
I have fixed it.